client/src/components/Modal/modal.css

```diff
@@ -0,0 +1,22 @@
+.backdrop {
+  background: rgba(0, 0, 0, 0.2);
+  backdrop-filter: blur(2px);
+  height: 100vh;
+  width: 100vw;
+  position: absolute;
+  top: 0;
+  left: 0;
+  display: flex;
```

Comment:
Wherever you give display flex, define flex property to all it's children. Don't rely on default values it can break